ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -29,6 +30,10 @@
  * class doesn't implement {@link ReactApplication}.
  */
 public class ReactActivityDelegate {
+
+  private final int REQUEST_OVERLAY_CODE = 1111;
```

Comment:
I can surely change the value, but I don't think there is any benefit of it being random. It doesn't need to be unique to other application just your own. So if your actvitiy ends up having multiple request codes its best to have them well defined, in order, and organized.
ie:
```
private final int MY_FIRST_CODE = 1111;
private final int MY_SECOND_CODE = 2222;
```
* Also it doesn't need to be 4 digits in length, just a common pattern I've noticed in the community.
* I'll update the name though, seems reasonable.